packages/react-composites/src/composites/ChatComposite/hooks/useSelector.ts

```diff
@@ -11,7 +11,7 @@ import { useSelectorWithAdaptation } from './useAdaptedSelector';
  */
 export const useSelector = <SelectorT extends (state: ChatAdapterState, props: any) => any>(
   selector: SelectorT,
-  selectorProps: Parameters<SelectorT>[1]
+  selectorProps?: Parameters<SelectorT>[1]
```

Comment:
Why? This doesn't seem like a change that should be part of this PR at all.

Reply:
selectorProps needs to be an optional prop.
It is an optional prop in call composite hooks as well.
This is a part of this PR since File Upload Button uses a selector using `useSelector` that can only be used after fixing this bug.